src/plugins/FileService/components/RecentFiles.tsx

```diff
@@ -0,0 +1,98 @@
+import { Button, List, ListItem, ListItemText, makeStyles, Typography } from '@material-ui/core'
+import { map } from 'lodash-es'
+import React from 'react'
+import { useHistory } from 'react-router'
+import { Image } from '../../../components/shared/Image'
+import { useI18N } from '../../../utils/i18n-next-ui'
+import { getUrl } from '../../../utils/utils'
+import type { FileInfo } from '../types'
+import { formatDateTime } from '../utils'
+
+const useStyles = makeStyles({
+    container: {
+        display: 'flex',
+        flexDirection: 'column',
+        width: 150,
+        paddingLeft: 10,
+    },
+    heading: {
+        fontSize: 14,
+        lineHeight: 1.1,
+        color: '#000',
+        userSelect: 'none',
+        marginTop: 0,
+        marginBottom: 10,
+    },
+    listing: {
+        flex: 1,
+    },
+    more: {
+        fontSize: 14,
+        lineHeight: 1.2,
+        color: '#2CA4EF',
+    },
+})
+
+const useItemStyles = makeStyles({
+    root: {
+        padding: 0,
+        paddingBottom: 10,
+        userSelect: 'none',
+        cursor: 'pointer',
+    },
+})
+
+const useItemTextStyles = makeStyles({
+    root: {
+        margin: 0,
+        marginLeft: 3,
+    },
+    primary: {
+        fontSize: 12,
+        lineHeight: 1.1,
+        color: '#14171A',
+        overflow: 'hidden',
+        whiteSpace: 'nowrap',
+        textOverflow: 'ellipsis',
+    },
+    secondary: {
+        fontSize: 10,
+        lineHeight: 1.2,
+        color: '#667886',
+    },
+})
+
+interface Props {
+    files: FileInfo[]
+    onMore(): void
+}
+
+export const RecentFiles: React.FC<Props> = ({ files, onMore }) => {
+    const { t } = useI18N()
+    const history = useHistory()
+    const classes = useStyles()
+    const itemClasses = useItemStyles()
+    const itemTextClasses = useItemTextStyles()
+    const onClick = (info: FileInfo) => () => {
+        history.push('/uploaded', info)
+    }
+    const renderItem = (file: FileInfo, index: number) => (
+        <ListItem classes={itemClasses} key={index} onClick={onClick(file)}>
+            <Image src={getUrl('/plugin/file-service/recent-file.svg')} width={32} height={32} />
```

Comment:
Please don't use this utility to render SVG. You can change the SVG into a TSX file and get a high-resolution icon. And if possible, please use the icon from [@material-ui/icons](https://material-ui.com/components/material-icons/) or [react-feather](https://feathericons.com/).